everware/user_wait_handler.py

```diff
@@ -50,7 +50,8 @@ def get(self, name, user_path):
                 html = self.render_template(
                     "spawn_pending.html",
                     user=current_user,
-                    need_wait=int(is_done)
+                    need_wait=int(is_done),
+                    __version__=__version__
```

Comment:
name of the argument starting with `__` seems a bit odd. Why not just `current_version` or `package_version` or `version`?